tox.ini

```diff
@@ -16,7 +20,9 @@ deps =
 install_command = pip install {opts} {packages} -c{toxinidir}/requirements.txt
 
 [testenv:py36-raw]
-passenv = TRAVIS
+passenv = TRAVIS CI
+setenv =
+    PIP_USE_DEPRECATED = legacy-resolver
```

Comment:
Should we create an issue to remove that?

Reply:
This is a problem with a dependency, but I'll still create one with details so we can keep track of when it's no longer needed.